src/scripts/main.js

```diff
@@ -1,3 +1,75 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+addRow.addEventListener('click', (evnt) => {
+  if (table.children[0].children.length >= 10) {
+    return;
+  }
+
+  removeRow.disabled = false;
+  table.children[0].append(table.querySelector('tr').cloneNode(true));
+
+  if (table.children[0].children.length >= 10) {
+    addRow.disabled = true;
+  }
+});
+
+removeRow.addEventListener('click', (evnt) => {
+  if (table.children[0].children.length <= 2) {
+    return;
+  }
+
+  addRow.disabled = false;
+  table.children[0].children[0].remove();
+
+  if (table.children[0].children.length <= 2) {
```

Comment:
better put table.children[0].children.length in a variable if you use it more than once